spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Contents.java

```diff
@@ -27,15 +27,15 @@
 /**
  * Class providing factory functions for BrandingDTO instances.
  */
-public final class Content {
+public final class Contents {
```

Comment:
If the goal here is to differentiate it from the constructed instances, we're better off calling these what they are: builders or factories. ContentBuilder, OwnerBuilder, etc..

Specifically for this class, "content" in English is an unnecessarily complex word when it comes to pluralization, and modern usage is narrowing on just using the unsuffixed variant for both singular and plural usage; since the strict guidelines for which to use are pretty subjective (whether or not the contents are countable, individual things, and what constitutes individuality or whether or not something is countable is highly context-sensitive).

As a native speaker who tries to follow proper grammar rules when it comes to selecting terminology or writing documentation, "contents," for our purposes, almost always feels clunky and reads awkwardly. Probably why modern usage is gradually deviating from the countable rule as more and more content moves to digital mediums.

tl;dr: English is a silly language.

Reply:
Fair enough.